content/css/concepts/display/display.md

```diff
@@ -46,6 +52,139 @@ The following values can be appplied to the `display` property:
 - `initial`
 - `inherit`
 
+## Differences between display values
+The display property in CSS allows you to control how elements are rendered on a webpage. Here are the commonly used display values and their descriptions:
+
+### 1. Display: block
+
+ Elements with display: block; are rendered as block-level elements. They create a line break after the element and occupy the full width of their parent container.
+```css
+.block-element {
+  display: block;
+}
+```
+
+### 2. Display: inline
+Elements with display: inline; are rendered as inline elements. They don't create line breaks and occupy only the necessary width and height based on their content.
+```css
+.inline-element {
+  display: inline;
+}
+```
+
+### 3. Display: inline-block
+
+  Elements with display: inline-block; are rendered as inline-level block elements. They behave like inline elements but allow you to set width, height, padding, and margins.
+```css
+.inline-block-element {
+  display: inline-block;
+}
+```
+### 4. Display: none
+
+ Elements with display: none; are not rendered and are completely hidden from the page. They don't take up any space in the document flow.
+```css
+.hidden-element {
+  display: none;
+}
+```
+### 5. Display: flex
+
+ Elements with display: flex; create a flex container, allowing you to build flexible and responsive layouts. Flex items inside the container can be positioned and resized using flexbox properties.
+```css
+.flex-container {
+  display: flex;
+}
+```
+### 6. Display: grid
+
+Elements with display: grid; create a grid container that enables you to define rows and columns, creating a two-dimensional grid system. Child elements inside the container can be placed in specific cells and adjusted for size and alignment.
+```css
+.grid-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 10px;
+}
+```
+
```

Comment:
Remove 22 blank lines. (Only one blank line between the code block and the following header.